recipes/cpr/all/conanfile.py

```diff
@@ -12,24 +12,20 @@ class CprConan(ConanFile):
     name = "cpr"
     description = "C++ Requests: Curl for People, a spiritual port of Python Requests"
     url = "https://github.com/conan-io/conan-center-index"
-    homepage = "https://whoshuu.github.io/cpr/"
+    homepage = "https://docs.libcpr.org/"
     license = "MIT"
-    topics = ("conan", "cpr", "requests", "web", "curl")
+    topics = ("cpr", "requests", "web", "curl")
     exports_sources = ["CMakeLists.txt", "patches/*"]
     generators = "cmake", "cmake_find_package"
     settings = "os", "arch", "compiler", "build_type"
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
-        "with_openssl": [True, False, "deprecated"],
-        "with_winssl": [True, False, "deprecated"],
         "with_ssl": ["openssl", "darwinssl", "winssl", _AUTO_SSL, _NO_SSL]
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "with_openssl": "deprecated",
-        "with_winssl": "deprecated",
```

Comment:
may it break consumers?

Reply:
Sure it will, from a warning to an error if they are using those options. But that is exactly the purpose of the intermediate _deprecation notice_, and it is why we wait some weeks before removing them.

Reply:
Yes, we already warned about removing deprecated options after some time. https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-recipe